internal/testgen/gen_bool_compare.go

```diff
@@ -26,6 +26,140 @@ func (g BoolCompareTestsGenerator) TemplateData() any {
 		ValidAssertions   []Assertion
 	}
 
+	ignoredAssertions := make([]Assertion, 0)
```

Comment:
https://github.com/uber-go/guide/blob/master/style.md#nil-is-a-valid-slice

or write
```go
var ignoredAssertions []Assertion
```

or use real capacity
```go
ignoredAssertions := make([]Assertion, 2*8+6)
// but! magic numbers + no make sense for this case anyway
```